pkg/controller/node/node_controller.go

```diff
@@ -182,10 +182,6 @@ func (ctrl *Controller) updateNode(old, cur interface{}) {
 	oldNode := old.(*corev1.Node)
 	curNode := cur.(*corev1.Node)
 
-	if oldNode.ResourceVersion == curNode.ResourceVersion {
```

Comment:
I'm not sure I understand this part. Is this also from moving from namespace to cluster level? Sorry, haven't actually used `client-gen` and the clientset stuff much. :)

Reply:
we are already suppressing syncs on nodes here https://github.com/openshift/machine-config-operator/pull/256/files/6bb04e10e49e11afcadf326017d3e1386f9fd684..1c73ebe8c2a5666a4ad9b73cc18fc3010346678a#diff-a44cd14a0381ca49bcbaaf967b3cc9b0R185

which is a  more concrete check.

So this check is not useful.